mk/cfg/x86_64-unknown-linux-uclibc.mk

```diff
@@ -0,0 +1,29 @@
+# x86_64-unknown-linux-uclibc configuration
```

Comment:
How do I run the exact tests you're referring to?

On Aug 4, 2016 3:52 PM, "Jorge Aparicio" <notifications@github.com> wrote:

> In mk/cfg/x86_64-unknown-linux-uclibc.mk
> <https://github.com/rust-lang/rust/pull/35242#discussion_r73590740>:
>
> > @@ -0,0 +1,29 @@
> > +# x86_64-unknown-linux-uclibc configuration
>
> altering their setup to build using the cargo build system.
> we'd just be wrapping the rust build script with make
>
> Yes, this is what I meant.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rust-lang/rust/pull/35242/files/64a1a13405b41722033f3dee0554e6148c7450c5#r73590740>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACUb3n9TXXxJrYnZ2inyDDYpV2KszmOLks5qckLigaJpZM4JbzyU>
> .
>